### PR TITLE
[3753] Remove wrong script from quest Sarkoth.

### PR DESCRIPTION
**Pull request by Nytanath**
Backport missing part of https://github.com/cmangos/tbc-db/commit/6edbb93cb8766c8634600ad39178e2a639408d8e

The quest [Sarkoth](https://classic.wowhead.com/quest=790/sarkoth) (entry: 790) currently has a wrong end script that changes his standstate from dead to kneel and then back to dead.

There actually is a script related to that NPC, and it's already scripted correctly (the on-start script for [Sarkoth](https://classic.wowhead.com/quest=804/sarkoth) (entry: 804)).

Basically the event was scripted twice, once on the end of the first step and once on the start of the second step. Only the second script is correct, the first should not exist.

To test:
- Create a Horde character.
- .go creature 3287
- .quest add 790
- .quest complete 790
- Turn in the quest, wait for the event, accept the new quest, wait for the event.

### CODE REVIEW

**Comment by AnonXS** [comment]
https://github.com/cmangos/tbc-db/commit/6edbb93cb8766c8634600ad39178e2a639408d8e

**Comment by Nytanath** [comment]
I see, seems like that was backported partially; the new script is same as that and the creature has same movement type and spawn dist. This PR completes the backport of that commit then, thanks for linking it!

**Comment by AnonXS** [comment]
https://github.com/cmangos/classic-db/commit/c750580ded6c0502b07779f1a1461c2713ab609d

**Comment by AnonXS** [comment]
https://github.com/cmangos/issues/issues/2034 this is where all got mixed up, we had the backport coming, the backport was not "enough" to fix it so completescript and startscript were exchanged. hence why its still wrong on other dbs.

**Comment by Nytanath** [comment]
Oh I see, was wondering why the script was still there even though the commit removing it was so old.

Can confirm there are no longer any issues with that script so the old one can be safely removed.

**Comment by Grz3s** [comment]
this quest is a curse.... or some ninja playin with us on it... 
here where it was fixed ,,, (have a look on my comment back then)
https://github.com/cmangos/wotlk-db/commit/bc59cb9db15a5a1244c944a1e92096fe3ecaa3b8
```
- q.790 'Sarkoth' - Script removed
- q.804 'Sarkoth' - Script updated
Why I'm fixing this X time already?
```
